assets/sass/patterns/atoms/list.scss

```diff
@@ -18,3 +18,7 @@ $prefixes: ("alpha-lower": "lower-latin", "alpha-upper": "upper-latin", "bullet"
     }
   }
 }
+
+.list--teaser {
+  @include padding($blg-space-small-in-px, "top");
```

Comment:
@nlisgo I'm not against the idea, I was saying that semantically the text didn't need to be wrapped any further than it was already. Sorry for the misunderstanding.

If wrapping the text in `<p>` gives what you need for vertical spacing, so then the `list--bullet` variant can just be injected as content then that's fine. That reduces the code changes to `pattern-library` considerably.

Will the text always be wrapped in a `<p>`, or only if there's a list following? If always (preferable, I think), then check if this affects the vertical spacing of teasers that don't have a list in the content (i.e. does the `<p>`add extra, unwanted space between these teasers?).